worker/runtime.go

```diff
@@ -15,47 +16,64 @@ type WasmRuntime struct {
 	mutex   sync.Mutex
 }
 
+// NewWasmRuntime initializes a new WasmRuntime instance.
 func NewWasmRuntime(ctx context.Context) *WasmRuntime {
 	return &WasmRuntime{
 		runtime: wazero.NewRuntime(ctx),
```

Comment:
I am OK that we leave Wazero as the current runtime, but I think we should definitely use [Wasmtime](https://wasmtime.dev/) in the future. It will not be embedded in Go, but installed on the Proplet node, so Proplet agent will consider that it already exist, and will use it to deploy Wasm module on it like CLI subprocess - probably via Go's [`exec`](https://pkg.go.dev/os/exec)